experimental/packages/opentelemetry-api-metrics/src/types/Metric.ts

```diff
@@ -71,9 +71,9 @@ export enum ValueType {
 
 /** The kind of aggregator. */
 export enum AggregationTemporality {
```

Comment:
Should this even be in the API? From what I understand, the temporality should be entirely handled in the SDK right?

Reply:
Yeah, previously it is being used in metric creation option and presented in metric descriptors. Now since the temporality is not determined at the metric interface anymore, I'd believe it is totally an SDK thing for metric streams.